src/components/plots/FrequencyPlot.jsx

```diff
@@ -1,17 +1,32 @@
-import React from 'react';
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
 import { Vega } from 'react-vega';
 import PropTypes from 'prop-types';
 
-import { generateSpec } from '../../utils/plotSpecs/generateFrequencySpec';
-import populateFrequencyData from './helpers/populateFrequencyData';
```

Comment:
This is also used in `plots and tables`. Check whether the `plots and tables` still working

Reply:
`FrequencyPlot` in `plots and tables` uses function in `generateFrequencyPlot`, so it is not affected by this change